### DISCUSSION TITLE
Thank you for bringing Linux users your video editor

### DISCUSSION THREAD

**Discussion by RafaelLinux** [olive-editor/olive#2101] [General]
I'm a long-time user of KDEnlive, which is really powerful because it allows you to do things like nest projects (without rendering) in other projects. However, every time I use Olive, I'm impressed by how fast and well it works, even though it still doesn't do as many things as KDEnlive.

Olive runs with amazing performance without configuring anything and even detects if you are using a non-native driver like "Nouveau". 

Thanks for continuing to show that developers make a difference.

Excellent work

**Comment by dairiki-ren** [comment]
I have devices on which the main three platforms are installed, i.e. Windows, MacOS and Linux (Pop!_OS, which is built on top on Ubuntu :) and I am really impressed by the features Olive has! For slapping some quick video footages together, this is perfect. But for advanced projects involving a lot of stuff, Olive has also got me covered. Now, I am not a professional editor and only make videos for fun, but I wish this project all the best to become competitive against other mainstream video editors! 
Cheers!

**Reply by dairiki-ren** [reply]
But actually, I kinda want to have a spoiler of what is going to come next...
